src/components/AdminComponenets/CreateSetForm.vue

```diff
@@ -0,0 +1,260 @@
+<template>
+  <div>
+    <h2>Nytt sett med kort</h2>
+    <div class="newDeckTopBar">
+      <div class="inputDeckName">
+        <label for="deckName">Navn på sett</label>
+        <input id="deckName" type="text" v-model="name" />
+      </div>
+      <button class="confirmButton" @click="saveCards" :disabled="!necessaryInformation">Lagre kort</button>
+    </div>
+    <div class="stepWrapper" v-if="editingBasicInfo">
+      <form class="newDeckForm">
+        <div class="inputElement">
+          <label>Er kortene i et par like eller forskjellige?</label>
+          <br />
+          <input type="radio" v-model="type" :value="typeEnums.IDENTICAL" />Like
+          <input type="radio" v-model="type" :value="typeEnums.DIFFERENT" />Forskjellige
+        </div>
+        <div class="inputElement">
+          <label for="cardsInPair">Antall kort i et "par"?</label>
+          <br />
+          <input
+            id="cardsInPair"
+            type="number"
+            min="2"
+            max="3"
+            v-model="cardsInPair"
+            :disabled="type == typeEnums.IDENTICAL"
+          />
+        </div>
+        <div
+          class="proceedButton"
+          @click="editingBasicInfo = false"
+          :style="setupInformation ? 'visibility: visible' : 'visibility: hidden'"
+        >
+          <font-awesome-icon :icon="['fas','arrow-right']" :style="{color: '#29b6f6'}" />
+        </div>
+      </form>
+    </div>
+    <div class="stepWrapper" v-else>
+      <div class="cardValuesTable">
+        <table>
+          <CardPairInput :cardsInPair="Number.parseInt(cardsInPair)" :addCardPair="addCardPair" />
+          <tr v-for="i in cards.length" :key="i">
+            <td>Par {{ i }}</td>
+            <td v-for="j in Number(cardsInPair)" :key="j+'a'">
+              <input v-model="cards[i-1][j-1]" type="text" />
+            </td>
+          </tr>
+        </table>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { addSet } from "../memo/sets.js";
+import { SetTypeEnum } from "../../common/enums.js";
+import CardPairInput from "./CardPairInput";
+
+export default {
+  components: {
+    CardPairInput
+  },
+  props: {
+    setCreated: {
+      type: Function
+    }
+  },
+  data() {
+    return {
+      name: "",
+      type: "",
+      cardsInPair: 2,
+      cards: [],
+      typeEnums: SetTypeEnum,
+      editingBasicInfo: true,
+      openTab: 1
+    };
+  },
+  computed: {
+    setupInformation() {
+      return this.type != "";
+    },
+    necessaryInformation() {
+      return this.name != "" && this.type != "";
```

Comment:
Referer tilbake til setupinformation:  

return this.name != "" && this.setupInformation()